### PR TITLE
[16.0][MIG] stock_free_quantity: Migration to 16.0 

### PR DESCRIPTION
**Pull request by EvaSForgeFlow**
Standard Migration to 16.0

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Code and functional review LGTM!

File: stock_free_quantity/__manifest__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "Stock Free Quantity",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.1",
```
Comment (AaronHForgeFlow) [inline stock_free_quantity/__manifest__.py]: `16.0.1.0.0`

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**